app/Console/Commands/GenerateThumbs.php

```diff
@@ -59,7 +60,11 @@ public function handle(): int
 			$amount = (int) $this->argument('amount');
 			$timeout = (int) $this->argument('timeout');
 
-			\Safe\set_time_limit($timeout);
+			try {
+				\Safe\set_time_limit($timeout);
+			} catch (InfoException) {
+				// We do nothing, set_time_limit will throw an error on tests.
```

Comment:
```suggestion
				// We do nothing; set_time_limit will throw an error, if larger than `max_execution_time` in `php.ini` due to security concerns
```

Reply:
Fixed in https://github.com/LycheeOrg/Lychee/pull/1336/commits/67962c91318de2146bf22694873143df13b68baf.